app/src/main/java/fi/metatavu/muisti/exhibitionui/persistence/model/Page.kt

```diff
@@ -24,8 +24,7 @@ data class Page (
     @TypeConverters(UUIDConverter::class)
     val pageId: UUID,
 
-    @NonNull
-    val name: String,
+    val name: String?,
```

Comment:
Changing this to be nullable would require migration, so change this back to nullable and use default name for unnamed pages.